docs/syntax/optional.md

```diff
@@ -991,15 +875,7 @@ The figure block must contain **only** two components; an image, in either Markd
 
 The title is optional and taken as the reference target of the figure:
 
-```md
-:::{figure-md} fig-target
-:class: myclass
-
-<img src="img/fun-fish.png" alt="fishy" class="bg-primary mb-1" width="200px">
-
-This is a caption in **Markdown**
-:::
-```
+::::{myst-example}
```

Comment:
I think the docs should probably use sth like:

```
![fishy](img/fun-fish.png)
```

For an example. Hopefully most people use the md image syntax, not the HTML one :)

Reply:
Done thanks!